gamemodes/fatkid/gamemode/maps/futuristicsubmarine/sh_init.lua

```diff
@@ -0,0 +1,13 @@
+  
+-- Copyright (C) 2021 Swamp Servers. https://github.com/swampservers/fatkid
+-- Use is subject to a restrictive license, please see: https://github.com/swampservers/fatkid/blob/master/LICENSE
+
+FATKID_BACKSTORY = [[
+<p>
+It was just an ordinary field trip in a nuclear submarine.

```

Comment:
Can you write something from scratch instead of just copying one and changing a few works